src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -2012,11 +2012,49 @@ public static void DCS_TypeWithXmlElementProperty()
     [Fact]
     public static void DCS_DifferentCollectionsOfSameTypeAsKnownTypes()
     {
-        Assert.Throws<InvalidOperationException>(() => { 
+        Assert.Throws<InvalidOperationException>(() => {
             (new DataContractSerializer(typeof(TypeWithKnownTypesOfCollectionsWithConflictingXmlName))).WriteObject(new MemoryStream(), new TypeWithKnownTypesOfCollectionsWithConflictingXmlName());
         });
     }
 
+    [Fact]
+    public static void DCS_MyPersonSurrogate_Stress()
```

Comment:
Why did you choose to add stress for surrogate feature instead of normal serialization? Is it because it reproduces the issue quicker?
I think there was a guideline stating that tests in corefx repository should run under x seconds. I recall that x=3. Does adding this stress test makes total running time longer significantly? It yes, I think it's better if we can use a test that is just minimal to reproduce the issue.